src/main/dotnet/ProjectFileParser_Tests/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSBuild" version="0.1.2" targetFramework="net40" />
```

Comment:
if msbuild package supports net40, what was the motivation for the hard dep on 4.5.2 ?